practice-02/src/index.html

```diff
@@ -17,18 +17,115 @@
+                                <div class="show-total-title">Manufacture</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30"
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/samsungnew-220x48-1.png" width="30"
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/OPPO42-b_5.jpg" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/logo-xiaomi-220x48-5.png" width="30"
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/vivo-logo-220-220x48-3.png" width="30"
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Realme42-b_37.png" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Nokia42-b_21.jpg" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Masstel42-b_0.png" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Itel42-b_54.jpg" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Mobell42-b_19.jpg" width="30" class="logo" />
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="show-total-item">
+                                <div class="show-total-title">Price</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item"> 2 - 4 million </a>
```

Comment:
why have space` 2 - 4 million `

Reply:
In the sample site I see spaces between them, but I have updated the space between the text prices

Reply:
We don't have any `the same site` here. Please update, 
![Uploading Task_7_Feat__Add_popover_for_filters_by_TheHienDuong_·_Pull_Request__60_·_Agility-Internship_Duong-The-Hien-Javascript.png…]()